libs/cdk/utils/directives/resize/resize.directive.ts

```diff
@@ -82,7 +83,7 @@ export class ResizeDirective implements OnChanges, AfterViewInit, OnDestroy {
     private _resizeSubscriptions = new Subscription();
 
     /** @hidden */
-    private _isRtl = false;
+    private readonly _direction$ = computed(() => (this._rtlService?.rtlSignal() ? -1 : 1));
```

Comment:
Having `$` for both signal and observable might be confusing code style. maybe we should ommit the `$` in case of signal and treat signals as functions?

Reply:
Usualy we use `$` for reactive properties. Signals are reactive, so I don't think it's an issue. Using `$` will flag property as something that is reactive (same as Observables).